### PR TITLE
Fix auth errors and fix broken alert logic

### PR DESCRIPTION
**Pull request by amoliksingh**

### CODE REVIEW

File: src/components/scenegraph/Toolbar.js [hunk 1]
```diff
@@ -318,9 +332,9 @@ export default class Toolbar extends React.Component {
       delete AFRAME.INSPECTOR.history.updates[id];
     }
 
-    if (deletedObjectsString.length > 0){
-      alert("The objects with the following ids were deleted: [ " + deletedObjectsString + " ]");
-    }
+    // if (deletedObjectsString.length > 0){
+    //   alert("The objects with the following ids were deleted: [ " + deletedObjectsString + " ]");
+    // }
```
Comment (ahmedhamodi) [inline src/components/scenegraph/Toolbar.js]: Intentional?

File: src/components/scenegraph/Toolbar.js [hunk 2]
```diff
@@ -361,9 +375,9 @@ export default class Toolbar extends React.Component {
     }
     this.setState({ objects });
     
-    if (changedObjectsString.length > 0){
-      alert("Changes to the following objects were made: [" + changedObjectsString + " ] were saved");
-    }
+    // if (changedObjectsString.length > 0){
+    //   alert("Changes to the following objects were made: [" + changedObjectsString + " ] were saved");
+    // }
```
Comment (ahmedhamodi) [inline src/components/scenegraph/Toolbar.js]: Intentional?

**Review by ahmedhamodi — COMMENTED** [review]
Couple of quick comments. If we are intentionally commenting it out, can prob delete it

**Review by ahmedhamodi — APPROVED** [review]

File: src/components/scenegraph/Toolbar.js [hunk 3]
```diff
@@ -268,7 +282,7 @@ export default class Toolbar extends React.Component {
             deletedObjectsString = deletedObjectsString + ", " + id.replace("-obj", "");
```
Comment (jaydhulia) [inline src/components/scenegraph/Toolbar.js]: This stuff isn't needed anymore right? This and `changedObjectsString`?

**Review by jaydhulia — APPROVED** [review]
Code changes look good. Are we good to merge?

**Pull request merged**